molecule/source/prepare.yml

```diff
@@ -28,6 +28,16 @@
         version: master
         update: yes
       with_dict: '{{ pulp_install_plugins }}'
+      when: "item.key.startswith('pulp')"
```

Comment:
How about when .git_repo isn't defined?

Because other plugins will need this too; some plugins are not under the pulp organization on GitHub.

Reply:
You can also use a default filter for the repo, and do this all in 1 task, dropping the when condition.
```
repo: '{{ item.value.git_repo | default( item.key| replace("-", "_") ) }}' 
```

Reply:
I left it as default for `pulp` repos, for repos outside pulp org I did the task below

Reply:
but this filter is cool, I will address it